app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1209,6 +1210,12 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
+    private val onboardScreenSection: Map<OnboardingScreenSection, Boolean> get() = try {
+        FxNimbus.features.onboardingcreen.value().sectionsEnabled
+    } catch (e: NimbusFeatureException) {
```

Comment:
We can remove this try catch since NimbusFeatureException isn't actually thrown. https://github.com/mozilla/application-services/issues/5085